develop/devguide/Connector/ConnectorHelp/Connector_help_pages.md

```diff
@@ -4,11 +4,13 @@ uid: Connector_help_pages
 
 # Connector documentation
 
-For every connector that is developed for DataMiner, 2 or more documentation pages should be made.
+For every connector that is developed for DataMiner, two or more documentation pages should be made.
 
-The first one, which we will call the [marketing page](#Marketing-page), should be appealing and focus on demonstrating the value the connector can bring to users. Comparing our Catalog to a real physical store, see this page as being the packaging of your product.
+The first page is the [marketing page](#marketing-page), which should be appealing and focus on demonstrating the value the connector can bring to users, similar to the packaging of a product in a store.
 
-The second one, which we will call the [technical page](#Technical-page), should be a subpage of the marketing one and contain further technical details. Comparing our Catalog to a real physical store again, see this page as being the technical notice of your product. This technical page details among others how the connections should be set up when you create an element with the connector, how the element should be used, etc. If a connector exports child connectors, a subpage should also be created for each of these child connectors.
+The second page is the [technical page](#technical-page), which should be a subpage of the marketing page containing further technical details. You could compare this to the technical label of a product in a store. The technical page details among others how the connections should be set up when you create an element with the connector, how the element should be used, etc.
```

Comment:
I'm not sure if the "technical label" is really a good comparision here. A technical label typically is stuck to a product and contains very limited info such as serial numbers and things like that. What I wanted to compare it to is the full paper document that you typically find within the box when you buy something from a store. It's typically almost like a book(let) containing plenty of information, how to use, what to be careful of, etc.

Reply:
other than the above, everything checks out to me.

Reply:
How about this alternative?
```suggestion
The second page is the [technical page](#technical-page), which should be a subpage of the marketing page containing further technical details. You could compare this to the instructions booklet included with a product in a store. The technical page details among others how the connections should be set up when you create an element with the connector, how the element should be used, etc.
```

Reply:
good enough for me. I would say let's release this and we can always further finetune it later on if required.